integration_test/tests/wallet.rs

```
@@ -329,6 +329,46 @@ fn wallet__list_received_by_label__modelled() {
     assert!(model.0.iter().any(|item| item.label == label));
 }
 
+#[test]
+fn wallet__import_multi() {
```

This test looks good in that it is testing using the three argument types of a descriptor, scriptPubkey and address. But this is testing the functionality of core. It would be nice to instead/as well add test cases that have Some `warning` and Some `error` to test the new structs added to `types`.

---

Actually the second request returns an error/warning, while the first and third/last request returns a success.

---

do you think it's better to use comment to indicate the request that passes and the one that fails in the test? as all is being returned using same structure, although different fields in the structure.

---

Yes, and could you add a comment saying why it should return an error.
For the error case can you add an assert that `warnings` and `error` do contain data.
Thanks.

---

Definitely, let me do that.